src/utils/specialWikisList.js

```diff
@@ -0,0 +1,40 @@
+const specialWikisList = [
```

Comment:
@dayllanmaza why are we hardcoding these?

Reply:
@davidbarratt What kaldari said. By doing this we remove ~80 wikis that will never be used in the timeline

Reply:
Ah, I see. I finally got the discussion on the ticket in my email.